Automattic/newspack-newsletters
# Assorted fixes & improvements

### All Submissions:






### Changes proposed in this Pull Request:



Closes #296 

### How to test the changes in this Pull Request:

1. #296
    1. Insert a Posts Inserter block, observe an option to append a "Continue reading…" link below posts
    2. Send a test newsletter or the campaign, confirm the link is in the email and that it links to the article
2. On `master`, observe setting a newsletter background color has no effect in the editor - observe the is does on this branch
3. Observe that meta values which influence rendering (styling, preview text) are correctly interpreted (this was subject of a refactor)
4. In block settings of Post Inserter, observe shorter settings labels (to avoid repetitive "Display…")
5. On `master`, observe a post caption is rendered in white color. Observe the issue fixed on this branch.

### Other information:
